### PR TITLE
docs: try rapidoc

### PR DESCRIPTION
**Pull request by alexgarel**
Trying [rapidoc](https://rapidocweb.com/api.html#slots)

to solve:
- #7883

### CODE REVIEW

File: docs/assets/api-rapidoc.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!doctype html> <!-- Important: must specify -->
```
Comment (alexgarel) [inline docs/assets/api-rapidoc.html:1]: Note to reviewer: download this file to try it locally it would work (appart the added menu urls)

**Comment by stephanegigandet** [comment]
Looks good. So this page would replace https://openfoodfacts.github.io/openfoodfacts-server/reference/api/ ?

image

**Comment by stephanegigandet** [comment]
The "parameters" list at the end is interesting, but it doesn't tell which path uses which parameter, so it's a bit weird.

**Comment by stephanegigandet** [comment]
Could you generate the api-v3 doc as well? I would like to see if it has the same issues as https://openfoodfacts.github.io/openfoodfacts-server/reference/api-v3/#tag/Write-Requests which does not show the product->packagings input field.

**Comment by alexgarel** [comment]
@stephanegigandet 

* at the moment it does not replace the old doc (but I will do it, as reverting is very easy as @teolemon made me notice)

* for the parameters, it's because I added the option to add the schema because I wanted to display "products" as an independent page (it could be handy for example to refer to that for csv export fields)

* I will add api-v3

**Comment by alexgarel** [comment]
@stephanegigandet do you want to validate (it's easy to roll-back if needed).

**Review by stephanegigandet — APPROVED** [review]
Looks good!

**Pull request merged**